File: Backend/fairshareapi/src/main/java/com/priyanshparekh/fairshareapi/auth/AuthResponseDto.java
package com.priyanshparekh.fairshareapi.auth;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class AuthResponseDto {
    String access_token;
    String refresh_token;
    int expires_in;
    Long id;
    String username;
    String email;
    String name;
    String profilePic;
    boolean deviceRegistered;
}

File: Frontend/FairShare/app/src/main/java/com/priyanshparekh/fairshare/auth/AuthResponseDto.kt
package com.priyanshparekh.fairshare.auth

data class AuthResponseDto(
    val access_token: String,
    val refresh_token: String,
    val expires_in: Int,
    val id: Long,
    val username: String,
    val email: String,
    val name: String,
    val profilePic: String,
    val deviceRegistered: Boolean
)
